_docs_integrate/request-one-time-consent-of-peer.md

```diff
@@ -37,7 +37,7 @@ To request an accept/reject decision from a peer to a free text, the ConsentRequ
 
 <!-- here is the description in detail  -->
 
-## Check your Requests validity
+## Check the Request's validity
```

Comment:
the validity belongs to the request, so it should be the Requests validity - right @tnotheis?

Reply:
Right. ;)

Reply:
Ahm, no, wrong. 😆